doc/config.toml

```diff
@@ -0,0 +1,35 @@
+# File generated by mage. DO NOT EDIT.
+# You can edit the template in config.toml.tmpl
+
+baseURL = "http://localhost/"
+languageCode = "en-us"
+title = "The Things Network Stack for LoRaWAN"
+theme = "hugo-theme-techdoc"
+
+enableGitInfo = true
+
+[frontmatter]
+  date = [":git", "date", "publishDate", "lastmod"]
+  lastmod = [":git", "lastmod", "date", "publishDate"]
+  publishDate = ["publishDate", "date"]
+  expiryDate = ["expiryDate"]
+
+[Params]
+  # Souce Code repository section
+  description = "The Things Network Stack for LoRaWAN is an open-source LoRaWAN network stack suitable for large, global and geo-distributed public and private networks as well as smaller networks."
+  github_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
+  version= "v3.0.3"
+  # Documentation repository section
+  # documentation repository (set edit link to documentation repository)
+  github_doc_repository = "https://github.com/TheThingsNetwork/lorawan-stack"
+  source_url = "https://github.com/TheThingsNetwork/lorawan-stack/blob/master/"
+
+[[menu.main]]
+  name = "Main"
```

Comment:
What is this title for, all of the documentation? Will there be other things here, and if so, what?

<img width="589" alt="Screenshot 2019-07-10 at 17 01 31" src="https://user-images.githubusercontent.com/13334001/60980237-62a33200-a334-11e9-972c-13620f58c82c.png">

Like, can we make `Main` into a Stack landing page and have the actual documentation in its own section?

Reply:
Removed, it's now just a page link